### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by ZeroZerg** [yasio/yasio#375] [Q&A] [answered]
We crashed at here in accident, the possibility is about 0.3%~0.5%. The version is 3.39.2

Stack is
yasio::inet::io_service::do_select(fd_set*, long long) + 1986
`
  int retval = ::select(this->max_nfds_, &(fdsa[read_op]), &(fdsa[write_op]), nullptr, &waitd_tv);
`
decltype(*(std::__ndk1::forward<yasio::inet::io_service*>(fp0)).*fp()) std::__ndk1::__invoke<void (yasio::inet::io_service::*)(), yasio::inet::io_service*, void>(void (yasio::inet::io_service::*&&)(), yasio::inet::io_service*&&) + 3442

Do you have any idea what cause the crash?

**Comment by halx99** [comment]
Needs more detail crash information, such as terminate signal number `SIGXXX`

**Comment by ZeroZerg** [comment, marked as answer]
Sorry to bother you.
It seems that the crash is happened at another thread. But the FireBase locked the crash at yasio.

**Reply by halx99** [reply]
ok, if you have any new question, feel free to report at here.
